sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select
+  "g"."name" as "genre",
+  count("cm"."actorId") as "films"
+from "genres" as "g"
+join "filmGenre" using ("genreId")
+join "castMembers" as "cm" using ("filmId")
+join "actors" as "a" using ("actorId")
+where "a"."actorId" = 178
```

Comment:
Assume the actor's id might change and only use the information you know up-front:

---

![image](https://user-images.githubusercontent.com/16143771/160180329-3ae48645-a136-41bb-906e-d5dea60dc62a.png)